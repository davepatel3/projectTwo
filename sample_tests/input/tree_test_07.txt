int
allocate
-60
0
allocate
85
0
allocate
-15
2
allocate
94
3
allocate
-23
2
ref_node
4
ref_node
1
allocate
76
0
allocate
87
2
allocate
36
5
set_parent
8
1
set_parent
7
0
remove
6
allocate
91
5
set_parent
3
5
set_parent
1
3
peek_nodes
remove
4
remove
7
allocate
-85
3
allocate
45
5
remove
1
set_parent
6
3
allocate
-12
7
allocate
75
0
allocate
-16
5
set_parent
4
2
allocate
63
5
remove
3
ref_node
6
remove
1
allocate
68
5
set_parent
7
6
allocate
16
3
allocate
35
9
remove
2
allocate
-87
0
remove
1
allocate
-59
0
peek_nodes
allocate
-2
3
allocate
-1
0
remove
10
quit
