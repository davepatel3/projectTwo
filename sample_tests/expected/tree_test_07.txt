allocate
1
allocate
2
allocate
3
allocate
4
allocate
5
ref_node
94
parent: 3
children: 
ref_node
-60
parent: 0
children: 
allocate
6
allocate
7
allocate
8
set_parent
set_parent
remove
allocate
6
set_parent
set_parent
peek_nodes
0: 0
1: -60
2: 85
3: -15
4: 94
5: -23
6: 91
7: 87
8: 36
remove
remove
allocate
4
allocate
7
remove
set_parent
allocate
8
allocate
1
allocate
9
set_parent
allocate
10
remove
ref_node
Node is recycled!
remove
allocate
6
set_parent
allocate
Node is recycled!
allocate
3
remove
allocate
1
remove
allocate
3
peek_nodes
0: 0
1: [recycled]
2: [recycled]
3: -59
4: [recycled]
5: [recycled]
6: [recycled]
7: [recycled]
8: [recycled]
9: [recycled]
10: [recycled]
allocate
9
allocate
10
remove
quit
